Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -132,13 +138,23 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate {
     var url: NSURL? {
         didSet {
             lockImageView.hidden = (url?.scheme != "https")
-            if let url = url?.absoluteString {
-                if url.hasPrefix("http://") ?? false {
-                    locationLabel.text = url.substringFromIndex(advance(url.startIndex, 7))
-                } else if url.hasPrefix("https://") ?? false {
-                    locationLabel.text = url.substringFromIndex(advance(url.startIndex, 8))
+
+            if let urlString = url?.absoluteString {
+
+                var truncatedURL: String
+                if let scheme = url?.scheme {
+                    truncatedURL = urlString.substringFromIndex(advance(urlString.startIndex, count(scheme) + count("://")))
```

Comment:
I also just noticed that on desktop we don't highlight the www. if it's visible where as in this branch we do. I'll update it to match desktop/android.